charts/sn-platform/templates/proxy/proxy-cluster.yaml

```diff
@@ -80,6 +80,9 @@ spec:
 {{- with .Values.proxy.extraSecretRefs }}
 {{ toYaml . | indent 4 }}
 {{- end }}
+    {{- if or .Values.auth.oauth.enabled .Values.auth.authentication.jwt.enabled }}
```

Comment:
I just found there is a duplicated code at lines 104-107.

https://github.com/streamnative/charts/blob/fixed/update-template-render/charts/sn-platform/templates/proxy/proxy-cluster.yaml#L104-L107